src/webapp/features/rabble_logo/rabbleLogo.tsx

```diff
@@ -0,0 +1,257 @@
+import React from 'react';
+import { useEffect } from "react";
+import styles from './rabbleLogo.module.css';
+
+const RabbleLogo = () => {
+  // this just display:none's the container after a countdown to diminish the chance of weird interactions with it hiding in the z-index.
+  useEffect(() => {
+    const disappear = setTimeout(() => {
+      let logoContainer = document.getElementById("disappear");
```

Comment:
This timer is working pretty well, but something tells me there's a more react-y way to do this, with useRef.  I'll think on it a bit; probably isn't necessary to change it now.

Reply:
ooh, okay! got it - I was at first just making sure I could do it, happy to improve next round!